pkg/networkservice/chains/usecase/usecase_test.go

```diff
@@ -234,50 +228,42 @@ func TestNSMGR_RemoteUsecase_BusyEndpoints(t *testing.T) {
 	require.NotNil(t, conn)
 	require.Equal(t, int32(2), atomic.LoadInt32(&counter.Requests))
 	require.Equal(t, 8, len(conn.Path.PathSegments))
+
+	// Close
+	e, err := nsc.Close(ctx, conn)
+	require.NoError(t, err)
+	require.NotNil(t, e)
+	require.Equal(t, int32(1), atomic.LoadInt32(&counter.Closes))
+
+	// Endpoint unregister
+	for i := 0; i < len(nsesReg); i++ {
+		_, err = s.domain.Nodes[1].EndpointRegistryClient.Unregister(ctx, nsesReg[i])
+		require.NoError(t, err)
+	}
 }
 
-func TestNSMGR_RemoteUsecase(t *testing.T) {
-	t.Cleanup(func() { goleak.VerifyNone(t) })
+func (s *nsmgrSuite) TestRemote() {
```

Comment:
I would be suggesting to focus on use cases. So each test for nsmgr suite should have the suffix Usecase and it should test a real use-case :)
```suggestion
func (s *nsmgrSuite) TestRemoteUsecase() {
```
Also apply this for other tests.